docs/install.md

```diff
@@ -8,7 +8,7 @@ and 45G of memory and a minimum of three nodes.
 
 You will also need to provide a docker compatible registry.
 
-## Configure your Registry
+## Configure your registry
 In order to make this install simple to walk through we recommend you
```

Comment:
is this install guide assuming that the user has a project id and cluster set up already?

Reply:
Good catch, I added some additional text explaining this.